static/js/activity.js

```diff
@@ -380,6 +380,7 @@ function focus_ping(want_redraw) {
     channel.post({
         url: '/json/users/me/presence',
         data: {status: (exports.has_focus) ? exports.ACTIVE : exports.IDLE,
+               ping_only: !want_redraw,
                new_user_input: exports.new_user_input},
         idempotent: true,
         success: function (data) {
```

Comment:
We should remove the TODO comment a few lines down, since this completes that TODO.